modules/nw-metallb-configure-address-pool.adoc

```diff
@@ -12,27 +12,29 @@ As a cluster administrator, you can add address pools to your cluster to control
 
 .Procedure
 
-. Create a file, such as `addresspool.yaml`, with content like the following example:
+. Create a file, such as `ipaddresspool.yaml`, with content like the following example:
 +
 [source,yaml]
 ----
-apiVersion: metallb.io/v1alpha1
-kind: AddressPool
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
 metadata:
   namespace: metallb-system
   name: doc-example
+  labels: <1>
+    zone: east
 spec:
-  protocol: layer2
   addresses:
   - 203.0.113.1-203.0.113.10
   - 203.0.113.65-203.0.113.75
 ----
+<1> This label assigned to the IPAddressPool can be referenced by the `ipAddressPoolSelectors` in the `bgpAdvertisement` CRD to associate the IPAddressPool with the advertisement. 
```

Comment:
ipAddressPoolSelectors is used in L2Advertisement also.

Reply:
`IPAddressPool` or IP address pool per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references